# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfsprogs/autoconf.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- xfsprogs-2.6.36.orig/configure.in	2005-08-01 16:20:46.000000000 +0200
+++ xfsprogs-2.6.36/configure.in	2005-08-09 13:32:05.770058750 +0200
@@ -1,6 +1,8 @@
 AC_INIT(include/libxfs.h)
 AC_CONFIG_HEADER(include/platform_defs.h)
 
+AC_PROG_CC
+
 AC_ARG_ENABLE(shared,
 [ --enable-shared=[yes/no] Enable use of shared libraries [default=yes]],,
 	enable_shared=yes)
@@ -30,6 +32,51 @@
 	test $enable_termcap = yes && libtermcap="-ltermcap",)
 AC_SUBST(libtermcap)
 
+# Generic macro, sets up all of the global packaging variables.
+# The following environment variables may be set to override defaults:
+#   DEBUG OPTIMIZER MALLOCLIB PLATFORM DISTRIBUTION INSTALL_USER INSTALL_GROUP
+#   BUILD_VERSION
+#
+AC_DEFUN([AC_PACKAGE_GLOBALS],
+  [ pkg_name="$1"
+    AC_SUBST(pkg_name)
+
+    . ./VERSION
+    pkg_version=${PKG_MAJOR}.${PKG_MINOR}.${PKG_REVISION}
+    AC_SUBST(pkg_version)
+    pkg_release=$PKG_BUILD
+    test -z "$BUILD_VERSION" || pkg_release="$BUILD_VERSION"
+    AC_SUBST(pkg_release)
+
+    DEBUG=${DEBUG:-'-DDEBUG'}           dnl  -DNDEBUG
+    debug_build="$DEBUG"
+    AC_SUBST(debug_build)
+
+    OPTIMIZER=${OPTIMIZER:-'-g'}        dnl  -O2
+    opt_build="$OPTIMIZER"
+    AC_SUBST(opt_build)
+
+    MALLOCLIB=${MALLOCLIB:-''}          dnl  /usr/lib/libefence.a
+    malloc_lib="$MALLOCLIB"
+    AC_SUBST(malloc_lib)
+
+    PKG_USER=${INSTALL_USER:-'root'}
+    pkg_user="$PKG_USER"
+    AC_SUBST(pkg_user)
+
+    PKG_GROUP=${INSTALL_GROUP:-'root'}
+    pkg_group="$PKG_GROUP"
+    AC_SUBST(pkg_group)
+
+    pkg_distribution=`uname -s`
+    test -z "$DISTRIBUTION" || pkg_distribution="$DISTRIBUTION"
+    AC_SUBST(pkg_distribution)
+
+    pkg_platform=`uname -s | tr 'A-Z' 'a-z' | sed -e 's/irix64/irix/'`
+    test -z "$PLATFORM" || pkg_platform="$PLATFORM"
+    AC_SUBST(pkg_platform)
+  ])
+
 AC_PACKAGE_GLOBALS(xfsprogs)
 AC_PACKAGE_UTILITIES(xfsprogs)
 
